src/orange/g4org/SolidConverter.cc

```diff
@@ -382,21 +382,46 @@ auto SolidConverter::ellipsoid(arg_type solid_base) -> result_type
 }
 
 //---------------------------------------------------------------------------//
-//! Convert an elliptical cone
+/*!
+ * Convert an elliptical cone
+ *
+ * Expressions for lower/upper radii were found by solving the system of
+ * equations given by [1]:
```

Comment:
Instead of this doxygen reference (it would probably be better to use the github mirror or the official doxygen docs) we usually just reference `\c G4EllipticalCone`.